### ISSUE TITLE
Crash when crafting quiver

### ISSUE THREAD

**Issue by crisdias** [CleverRaven/Cataclysm-DDA#61651]
### Describe the bug

As soon as I start to craft a quiver, the game crashes with error message above. `<space>` does not send me back to the game, I have to use `<I>`. After that, I get prompted with the ingredient selection box and crafting proceeds normally.

### Steps to reproduce

1. Enter craft menu. `<&>`
2. Start crafting quiver

### Expected behavior

No crashes. :-)

### Versions and configuration

```
DEBUG    : Unable to trim string '' to width 0.  Returning empty string.

 FUNCTION : trim_by_length
 FILE     : D:\a\Cataclysm-DDA\Cataclysm-DDA\src\output.cpp
 LINE     : 275
 VERSION  : cd5924e

```

Windows 10, experimental, tiles, version 2022-10-12-1638

**Comment by NetSysFire** [comment]
fwiw, this is not a crash. It is a nasty error but the game does not crash. I can confirm this issue though.

![image](https://user-images.githubusercontent.com/59517351/195478053-27a419d2-dc24-4f6a-96b0-6f872bb6e09e.png)

**Comment by ZeroInternalReflection** [comment]
I'll double-check this soon, but I'm reasonably confident that this is a duplicate of #61591, and should be fixed by #61626.

I'm still trying to figure out something useful to say in the entry field though.

**Comment by NetSysFire** [comment]
Closing as duplicate.
